uPortal-portlets/src/main/java/org/apereo/portal/portlets/portletadmin/PortletAdministrationHelper.java

```diff
@@ -268,7 +268,9 @@ private PortletDefinitionForm createNewPortletDefinitionForm() {
      */
     public PortletDefinitionForm savePortletRegistration(
             IPerson publisher, PortletDefinitionForm form) throws Exception {
-        logger.warn("BEACH TEST:  In savePortletRegistration for: "+form.getPortletName());
+        if (this.logger.isInfoEnabled()) {
```

Comment:
:question: Should this be controlled by a global log level rather than a per class property?

Reply:
My understanding of this construct is it helps JVM performance.  The global log level still controls what's written out to the log files.